core/src/components/progress-bar/progress-bar.scss

```diff
@@ -0,0 +1,208 @@
+@import "../../themes/ionic.globals";
+// Progress bar
+// --------------------------------------------------
+:host {
+    /**
+     * @prop --buffer-background: Background of the buffer bar
+     * @prop --buffer-color: Color of the buffer circles
+     * @prop --buffer-size: Size of the buffer circles
+     * @prop --buffer-speed: Speed of the buffer circles
+     */
+    --buffer-background: rgba(0, 0, 0, .30);
+    --buffer-color: currentColor;
+    --buffer-size: 10px;
+    --buffer-speed: 450ms;
```

Comment:
Not really a speed, but a duration. I think it's better to be more explicit here, how about:
`--buffer-animation-duration`?